### DISCUSSION TITLE
purpose of web/www/horas/OP1967

### DISCUSSION THREAD

**Discussion by happy-lisper** [DivinumOfficium/divinum-officium#3757] [General]
Laudétur Jesus Christus!

I am new contributor to the project and mostly interested in Dominican Breviary. After reviewing the code and "database" (which is simple in the principle and complicated with details) I have 2 questions:

1. what is the purpose of web/www/horas/OP1967? I mean how, when, where in the perl code it is used.
2. in general - besides translations of course - what is missing as far as main Latin text goes? (I found the answer in previous thread)

Thank you

**Comment by mbab** [comment]
1. This is not used in code at all.
2. Probably almost all differences with Roman Breviary are not implemented - apart skeleton for Prima and Completorium.

**Reply by happy-lisper** [reply]
@mbab  re 1, thanks for the confirmation, this is what I thought.

So in `./Latin/Ordinarium`, I noticed there are postfixes for all Office Hours.

- none: it is Roman Latin
- `OP`: Dominican
- `M`: Monastic
- `1960`: not sure?

Is the main mechanism to select a rite?

**Reply by mbab** [reply]
Correct.
* none: is for Tridentine & Divino Afflatu
* `1960` is used by both Rubrics 1960

In fact I have plan to integrate those files in _one per hour_, and also remove `Ordinarium/` from vernaculars.
